arangodb-net-standard.Test/CollectionApi/CollectionApiClientTest.cs

```diff
@@ -217,5 +217,37 @@ public async Task GetCollectionCountAsync_ShouldSucceed()
             }));
             Assert.Equal(HttpStatusCode.NotFound, exception.ApiError.Code);
         }
+
+        [Fact]
+        public async Task GetCollectionAsync_ShouldSucceed()
+        {
+            await GenerateCollections(new string[] { "MyTempCollection" });
+
+            var collection = await _collectionApi.GetCollectionAsync("MyTempCollection");
+
+            Assert.Equal("MyTempCollection", collection.Name);
+        }
+
+        [Fact]
+        public async Task GetCollectionAsync_ShouldThrowNotFound()
```

Comment:
We don't have developer docs like this yet but prevailing convention for test names should be:

```
TestName_Should<ExpectedResult>_When<ConditionOrContext>
```
So let's call this something like: `GetCollectionAsync_ShouldThrow_WhenCollectionNotFound`.  (This convention exists for Centro so it's what we've been using here as well)